support/templates/agenda-prepublish.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="agendapunten" prefix={{prefixes}}>
+    <span property="http://data.vlaanderen.be/ns/besluit#isGehoudenDoor"
+          resource={{zitting.bestuursorgaan}}> </span>
```

Comment:
do make sure to wrap element attributes in quotes though. So 
```resource="{{zitting.bestuursorgaan}}" ```

Reply:
huh I thought that wasnt needed but I just tested and you're right
Weird that it didnt cause more trouble

Reply:
it seems ember corrects this when rendering, because it is rendered correctly in the frontend indeed.